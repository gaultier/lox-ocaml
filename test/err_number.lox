print 1.; // expect: 1:8:Trailing `.` in number not allowed: `1.`
