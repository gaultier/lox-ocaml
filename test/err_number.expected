1:8:Trailing `.` in number not allowed: `1.`
